x-pack/plugins/task_manager/server/routes/health.ts

```diff
@@ -95,13 +115,48 @@ export function healthRoute(
   router.get(
     {
       path: '/api/task_manager/_health',
+      // Uncomment when we determine that we can restrict API usage to Global admins based on telemetry
+      // options: { tags: ['access:taskManager'] },
       validate: false,
     },
     async function (
       context: RequestHandlerContext,
       req: KibanaRequest<unknown, unknown, unknown>,
       res: KibanaResponseFactory
     ): Promise<IKibanaResponse> {
+      // If we are able to count usage, we want to check whether the user has access to
+      // the `taskManager` feature, which is only available as part of the Global All privilege.
+      if (usageCounter) {
+        const clusterClient = await getClusterClient();
+        const { body: hasPrivilegesResponse } = await clusterClient
+          .asScoped(req)
+          .asCurrentUser.security.hasPrivileges({
+            body: {
+              application: [
+                {
+                  application: `kibana-${kibanaIndexName}`,
+                  resources: ['*'],
+                  privileges: [`api:${kibanaVersion}:taskManager`],
```

Comment:
Do you know how I can simulate someone with this permission that isn't also a superuser? I can't find this level of granularity in the roles UI

Reply:
I don't believe there's a way in the UI to just add this privilege in the role. This is only included in the Global All set of privileges. We originally wanted to restrict the API to just superusers but are worried about the SDH implications where we ask for the output for debugging purposes. In the future if we add a task manager UI, we can add specific privileges for that feature and expand access from just superusers to that privilege as well.

Reply:
Gotcha, thanks for the clarification!

Reply:
Because of the changes in `x-pack/plugins/security/server/authorization/privileges/privileges.ts`, a "Global admin" (_not_ superuser) will have this privilege.

"Global admin" is shorthand for a user who holds the "All" base privilege in "* All spaces".

Reply:
"Global admin" users are those who hold the "All" privilege for "* All spaces" (which is synonymous with the "global resource").

These users are _not_ superusers. The `kibana_admin` built-in role is a "Global admin", you can also create a custom role like so:

![image](https://user-images.githubusercontent.com/5295965/143137164-98aad819-3318-44fe-8b72-ae8a0711d063.png)

Reply:
@jportner Yep! I tested creating a custom role with Global admin.